release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -121,6 +121,44 @@ io:println(nameText); // "DanBrown"
 
 ##### Language Server
 
+##### Ballerina Shell
+
+Ballerina Shell now supports redefining module-level declarations and variable declarations. `/remove NAMES` command can be used to remove one or multiple declarations from snippet memory.
```

Comment:
Shall we split this into two? First mention redefining then add the snippet and then add the content on remove and the snippet?